components/event-processor/org.wso2.carbon.event.processor.admin/src/main/java/org/wso2/carbon/event/processor/admin/EventProcessorAdminService.java

```diff
@@ -185,7 +185,7 @@ public int compare(Object o1, Object o2) {
                     }
                 });
 
-                if (!isDistributedProcessingEnabled()){
+                if (!isDistributedProcessingEnabled()) {
                     configurationDtos[0].setDeploymentStatus("not-distributed");
```

Comment:
@imesh 
Thank you for pointing out. Moving "not-distributed" to a constant is fixed in PR:https://github.com/wso2/carbon-event-processing/pull/183 . 

The reason for updating only the zeroth element here is, with the current implementation we use "configurationDtos[0].deploymentStatus" as a flag which is being set to indicate whether CEP server is started in distibuted mode or not. We have used this approach to avoid two admin service calls which might adversly affect loading the UI. Please point out if this approach has to be changed.